### PR TITLE
Adds a suit cycler to the tramp freighter

### LINKED ISSUE #17935: [Bug]: No Suit Cycler on the Tramp Freighter
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Third party ship

### Describe the issue

They forgot to readd the suit cycler when they remapped the freighter vessel.

image

### How to reproduce

_No response_

### Round ID

_No response_

### Staff ckey

ReadThisNamePlz

### PR DESCRIPTION
**Pull request by LynxSolstice**
Adds a suit cycler to the tramp freighter, which is required to use voidsuits as a non-human species, and was overlooked in the remap of the vessel.

Fixes #17935

### CODE REVIEW

**Comment by LynxSolstice** [comment]
!review

**Review by FluffyGhoster — APPROVED** [review]

**Comment by RustingWithYou** [comment]
The suit cycler mapped in hasn't had its req_access variable changed, meaning that the tramp freighter crew will be unable to use it.

File: maps/away/ships/tramp_freighter/tramp_freighter.dmm [hunk 1]
```diff
@@ -3831,6 +3831,7 @@
 /obj/structure/cable/green{
 	icon_state = "1-4"
 	},
+/obj/machinery/suit_cycler,
```
Comment (RustingWithYou) [inline maps/away/ships/tramp_freighter/tramp_freighter.dmm:3834]:
```suggestion
/obj/machinery/suit_cycler{
	req_access = null
	},
```

Comment (DreamySkrell) [reply maps/away/ships/tramp_freighter/tramp_freighter.dmm:3834]: this

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**